### PR TITLE
[Preg:: issue] udiff output fix

### PR DESCRIPTION
**Pull request by SpacePossum**
replaces https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4082

### CODE REVIEW

**Comment by SpacePossum** [comment]
not sure how to resolve this `Preg::` thing

**Comment by kubawerlos** [comment]
@SpacePossum modify `array_filter` callback in https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.13.1/tests/AutoReview/ProjectCodeTest.php#L366

**Comment by keradus** [comment]
@SpacePossum , can you elaborate on that label?
what help you need? 
I believe suggestion of @kubawerlos is perfect to finish this PR!

**Comment by keradus** [comment]
:man_facepalming:

File: src/Differ/DiffConsoleFormatter.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ function ($matches) {
 
                         return sprintf($lineTemplate, $string);
                     },
-                    Preg::split('#\R#u', $diff)
+                    preg_split('#\R#', $diff)
```
Comment (keradus) [inline src/Differ/DiffConsoleFormatter.php]: please add a warning to not change this into `Preg::...` and why

Comment (kubawerlos) [reply src/Differ/DiffConsoleFormatter.php]: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4107

**Comment by SpacePossum** [comment]
closing in favor of https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4107

**Pull request closed without merging**